tsconfig.json

```diff
@@ -1,8 +1,8 @@
 {
   "compilerOptions": {
-    "outDir": "./build",
+    "outDir": "./build/src/",
```

Comment:
Can you explain why you need to add `src/`?

Reply:
Because I copy the package.json into build/src

Reply:
Oh, and otherwise everything would end up in the build directory and that's breaking compassql.